runtime/lua/vim/lsp/protocol.lua

```diff
@@ -923,4 +923,271 @@ function protocol.resolve_capabilities(server_capabilities)
   return server_capabilities
 end
 
+--- LSP methods. Generated by lsp_types.lua, keep at end of file.
```

Comment:
the "generated by..." does not belong in user-facing docs, so it should have only two hypens `--`. See if this works...

```suggestion
-- Generated by lsp_types.lua, keep at end of file.
--- LSP methods.
```

Reply:
not work lua_ls still parse `--`